editor/plugins/polygon_2d_editor_plugin.cpp

```diff
@@ -1239,6 +1264,11 @@ void Polygon2DEditor::_uv_draw() {
 		}
 	}
 
+	if (uv_edit_mode[1]->is_pressed()) {
```

Comment:
Unrelated, but the modes should get an enum.

Reply:
Yeah this code is a bit all over the place and there's some confusing terminology for modes, actions and menu items. I've outlined some refactoring work which I want to do in godotengine/godot-proposals#8209 (input on those points would be appreciated).